apps/pie-docs/src/assets/styles/components/_categorisedIconList.scss

```
@@ -16,6 +16,11 @@ $icon-preview-size: 64px;
     list-style-type: none;
 }
 
+.c-categorisedIconList-select {
+    display: inline-block; 
+    margin-top: var(--dt-spacing-d);
```

sorry i forgot to mention but we use CSS logical properties to set margin and padding values. this is to allow us to make layouts adapt better to different writing directions (like left-to-right and right-to-left). 

```suggestion
    margin-block-start: var(--dt-spacing-d);
```


https://css-tricks.com/css-logical-properties-and-values/